prime-router/metadata/hl7_mapping/common/specimen.yml

```diff
@@ -64,6 +64,17 @@ elements:
     value: [ '""' ]
     hl7Spec: [ '%{hl7SpecimenFieldPath}-11' ]
 
+  - name: specimen-collection-amount-quantity
+    value: [ '%resource.collection.quantity.value' ]
+    hl7Spec: [ '%{hl7SpecimenFieldPath}-12-1' ]
+
+  - name: specimen-collection-amount-units
+    condition: '%context.collection.quantity.where(extension(%`rsext-cq-units`).value).exists()'
+    resource: '%resource.collection.quantity.extension(%`rsext-cq-units`).value'
+    constants:
+      cweFieldPath: '%{hl7SpecimenFieldPath}-12-2'
+    schema: ../datatypes/codeableConcept/CWE
```

Comment:
Thoughts on extracting this into a `datatypes/Quantity/CQ.yml` and just passing `%resource.collection.quantity` through to that? Not sure how common this will be, or therefore how reusable that would be. Consider it optional, this works fine too.

Reply:
added Quantity/CQ.yml since it will be used in two places. Good rec!